source/features/hide-empty-meta.tsx

```diff
@@ -2,8 +2,10 @@ import select from 'select-dom';
 import features from '../libs/features';
 
 function init(): void {
-	if (select.exists('.repository-meta em') && !select.exists('.js-edit-repo-meta-button')) {
-		select('.repository-meta')!.style.display = 'none';
+	const emptyDescription = select('.repository-content > :not(#readme):not(.commit-tease) em');
```

Comment:
There are a lot of other children in `.repository-content`, we can't go by exclusion

<img width="513" alt="Screenshot 2019-05-14 at 00 22 06" src="https://user-images.githubusercontent.com/1402241/57637603-6a698300-75de-11e9-9e7d-cb50cd63c886.png">

It should probably start with `#repo-meta-edit` or maybe just be `#repo-meta-edit em`. Double-check

Reply:
I also think the selector is different if you have rights to edit

Reply:
> There are a lot of other children in `.repository-content`, we can't go by exclusion

But usually commit list an the the readme are the only likely ones to contain an `em`.

-----------------

> It should probably start with `#repo-meta-edit` or maybe just be `#repo-meta-edit em`. Double-check

Targeting `#repo-meta-edit em` fails for repos with no permissions, the existence of `#repo-meta-edit` is the condition to bail.

Reply:
> I also think the selector is different if you have rights to edit

`#repo-meta-edit` only exists if you have permission, else its just `em` inside a "`div`-tree-with-utility-class" hell.

Reply:
True, actually because it's so high in the dom we might get away with just `.repository-content em` because `select` only gets the first one

Reply:
<img width="361" alt="Screenshot 2019-05-14 at 00 38 15" src="https://user-images.githubusercontent.com/1402241/57638552-9554d680-75e0-11e9-837a-7477f4e7988c.png">

Yeah it's definitely the first one

Reply:
But it should drop `d-flex`, not just em, otherwise there's some space left behind